README.md

```diff
@@ -107,9 +107,9 @@ This will create a `modulizer_workspace` directory and checkout the repos and th
 
 Setting the import style allows you to set whether JavaScript imports are specified by npm package name, or relative file path. Importing specifiers that use package names are easier for third-party packages to work with, but unlike paths they currently can not run natively on the web. Defaults to "path".
 
-#### `--add-import-path`
+#### `--remove-import-meta
```

Comment:
I would lean towards keeping `--add-import-path` and just changing the default to `true`.